kuwala/core/canvas/src/components/Modals/NodeConfigModal.js

```diff
@@ -0,0 +1,502 @@
+                setSchema(populatedSchema)
+            }
+        }
+        setIsSchemaLoading(false)
+    }
+
+    const renderSelectedSourceHeader = () => {
+        if (!selectedElement) {
+            return <></>
+        } else {
+            return (
+                <div className={'flex flex-row'}>
+                    <div className={'flex flex-col items-center'}>
+                        <div
+                            className={'flex flex-col justify-center items-center bg-white rounded-xl drop-shadow-lg relative p-4 w-24 h-24'}
+                        >
+                            <img
+                                src={selectedElement.data.dataSource.logo}
+                                style={{height: 48, width: 48}}
+                                draggable={false}
+                            />
+                            <span className={'mt-1 text-sm'}>{selectedElement.data.dataSource.name}</span>
+                            <div
+                                className={`
+                                    absolute right-0 top-0 p-1 border rounded-full w-5 h-5 -mr-2 -mt-2
+                                    ${selectedElement.data.dataSource.connected ? "bg-kuwala-green" : "bg-red-400"}
+                            `}
+                            />
+                        </div>
+                    </div>
+
+                    <div className={'flex flex-col ml-6 space-y-2 bottom-0 justify-end mb-2'}>
+                        <span className={'px-3 py-1 bg-kuwala-light-green text-kuwala-green font-semibold rounded-lg w-36'}>
+                            Data blocks
+                        </span>
+
+                        <div className={'flex flex-row items-center'}>
+                            <label className={'font-semibold'}>Name:</label>
+                            <input
+                                type="text" name="name" value={`${selectedElement.data.dataBlocks.name || '-'}`} disabled={true}
+                                className={`
+                                    form-control
+                                    block
+                                    w-full                                   
+                                    ml-2
+                                    px-2
+                                    py-0.5
+                                    text-base
+                                    font-light
+                                    text-gray-700
+                                    bg-gray-100 bg-clip-padding
+                                    border border-solid border-kuwala-green
+                                    rounded-lg
+                                    transition
+                                    ease-in-out
+                                    m-0
+                                    focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none
+                                `}
```

Comment:
I feel we should move these styling configurations to separate files and then use them like `className={styles.nameHeader}`. Would make the code more readable IMO. Observed this in other PRs as well.

Reply:
This one will took lots of effort, and i think it is best done when we have a components design guideline ready

Reply:
for now i leave it as is for existing components, and make sure to encapsulate the styling when creating a component later.